lib/core/versions/latest/BatchWriter.ts

```diff
@@ -27,11 +28,14 @@ import ValueTimeLockVerifier from './ValueTimeLockVerifier';
  * Implementation of the `IBatchWriter`.
  */
 export default class BatchWriter implements IBatchWriter {
```

Comment:
For some reason branch coverage for BatchWriter is no longer 100%, bring it back please!